requirements.txt

```diff
@@ -0,0 +1,3 @@
+cryptography==3.3.1
```

Comment:
I looked a bit at the syntax for semver operators in `requirements.txt`, but couldn't quite figure it out. The `==` operator just pins to a specific version, right? Is there something we can do here that moves away from just pinning that would allow using cryptography 3.x, fake-useragent 0.1.x and requests 2.x? (I'm not that active in the Python ecosystem these days.) Presumably those releases would be pretty compatible, certainly for the pretty minimal API surface used here; and I don't really want to deal with PRs just to bump the pinned versions -- even just the named dependencies would be better for this particular use IMO.